### ISSUE TITLE
 Yarn fails to install (Exit code: 128)

### ISSUE THREAD

**Issue by thenameisajay** [yarnpkg/yarn#9045]
**Issue Title:** Unable to fetch packages from remote repository during yarn install

**Description:**
When attempting to run `yarn install` in the project directory, the process fails to fetch packages from a remote repository with the following error message:

```
yarn install v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
error Command failed.
Exit code: 128
Command: git
Arguments: ls-remote --tags --heads ssh://git@github.com/googleapis/nodejs-vertexai.git
Directory: /vercel/path0
Output:
Host key verification failed.
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Behavior:**
The `yarn install` command should successfully resolve and fetch packages from the remote repository without any errors.  

Seen behaviour : Works on local but on production it fails to fetch the package

**Environment:**
- Operating System:  [macOS Ventura]
- Yarn version: [e.g., 1.22.19]

**Comment by clintmod** [comment]
Host key verification failed means that you have an entry in your `$HOME/.ssh/known_hosts` file that doesn't match what github.com is returning.

Here's a post on how you can fix it https://serverfault.com/questions/321167/add-correct-host-key-in-known-hosts-multiple-ssh-host-keys-per-hostname.

TLDR: Add the correct key returned by 
```
 ssh-keyscan -t rsa github.com
```
to your known_hosts file
